app/src/common/shared/com/igalia/wolvic/browser/LoginStorage.kt

```diff
@@ -23,17 +22,18 @@ class LoginStorage(
     init {
         EngineProvider.getOrCreateRuntime(context).setUpLoginPersistence(places.logins)
         GlobalScope.launch(Dispatchers.IO) {
-            try {
-                storage.value.warmUp()
-            } catch (e: InvalidKeyException) {
-                // Login database sometimes gets corrupted for unknown reasons. This has been reported
-                // to mozilla components in the past (https://github.com/mozilla-mobile/android-components/issues/6681
-                // or https://github.com/mozilla-mobile/fenix/issues/15597) but it was never really
-                // fixed so clients have to deal with that. The only thing we could do is to delete
-                // the database file. We cannot just use wipe()/wipeLocal() because those calls also
-                // try to use the connection to the DB and thus they'll crash as well.
-                places.clearLoginsDatabaseUglyHack()
-            }
+            storage.value.warmUp()
+//            try {
+//                storage.value.warmUp()
+//            } catch (e: InvalidKeyException) {
+//                // Login database sometimes gets corrupted for unknown reasons. This has been reported
+//                // to mozilla components in the past (https://github.com/mozilla-mobile/android-components/issues/6681
+//                // or https://github.com/mozilla-mobile/fenix/issues/15597) but it was never really
+//                // fixed so clients have to deal with that. The only thing we could do is to delete
+//                // the database file. We cannot just use wipe()/wipeLocal() because those calls also
+//                // try to use the connection to the DB and thus they'll crash as well.
+//                places.clearLoginsDatabaseUglyHack()
+//            }
```

Comment:
We don't keep commented code.

This caused many breakages in the past. Mozilla mentioned several times that it was already fixed, but it reapeared again and again. Anyway I'm fine with trying it again after this upgrade.

BTW don't forget to delete clearLoginsDatabaseUglyHack()